{{cookiecutter.book_slug}}/.gitlab-ci.yml

```diff
@@ -31,3 +31,9 @@ pages:
       - public
   only:
     - master
+    
+workflow:
+  rules:
+    - if: $CI_COMMIT_REF_NAME =~ /-wip$/
```

Comment:
I don't use GitLab CI myself but understand it at a high level. To be clear, the first rule is somewhat of a personal choice, in that CI won't trigger for references with "-wip"? I wonder if something else, such as "-dev", would be more universal here? Or is "-wip" the norm in the GitLab community? I'm fine with merging this in, just want to clarify and get your thoughts. Thanks for the PR!

Reply:
Hi, That's my understanding as well but I've never used it. I copied both rules from docs (?) a while ago but TBH have never used `-wip`.  Feel free to only leave the second rule or change to `dev`.

Reply:
Thanks, I'll leave as is with a comment. I think it's nice to have in as it's a heads-up to new users that there are more advanced  configuration options possible. While I'm here, I'm also going to change the default branch to "main" from "master" - will merge shortly, cheers